README.md

```diff
@@ -18,6 +18,37 @@ Now start ZeroTier:
 
 ## Compiling from Sources
 
+### Feed
+A working OpenWRT/LEDE build enviroment is expected (else see below).
+
+Put this line in your feed definition (eg. feeds.conf.default)
+```
+src-git zerotier https://github.com/muebau/zerotier-openwrt.git
```

Comment:
heh, you've put in your repository there. I guess we should move the whole package to https://github.com/openwrt/packages anyway.